### ISSUE TITLE
If you let the app run for a while, all screens go blank

### ISSUE THREAD

**Issue by advayDev1** [WorldHealthOrganization/app#260]
Repro:
Start the app on the web browser
Leave it on, for example, on the menu or about page
Come back to it

Expected: Screen is still there
Actual: Screen has gone blank

````
8.16034357.chunk.js:formatted:15567 DOMException: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at cs (https://preview.whoapp.org/static/js/8.16034357.chunk.js:2:294570)
    at pu (https://preview.whoapp.org/static/js/8.16034357.chunk.js:2:307761)
    at t.unstable_runWithPriority (https://preview.whoapp.org/static/js/8.16034357.chunk.js:2:328501)
    at zi (https://preview.whoapp.org/static/js/8.16034357.chunk.js:2:252716)
````

**Comment by epicfaace** [comment]
Perhaps this has to do with this? https://github.com/ionic-team/ionic/issues/18782

**Comment by advayDev1** [comment]
https://github.com/ionic-team/ionic/issues/18782 looks promising

Maybe make sure everything has a unique key that is a hash?

**Comment by advayDev1** [comment]
Have a fix for one cause of this

**Comment by epicfaace** [comment]
I don't think 325512579d498de21e713e92c07cf5cc9bc244f4 fixes a race case like in #264, because there's no analogous situation in which there is a boolean check for `flow` that is supposed to evaluate to false but evaluates to true when `flow` is empty.

Indeed, the only place where `flow` is used / checked is here, which would behave normally (i.e. not render) if `flow` were set to `{}` by default:
https://github.com/WorldHealthOrganization/app/blob/628f125369a62cff4c9ba27ded35b6a36fe578b8/client/ionic/src/components/Carousel.tsx#L76-L78

**Comment by advayDev1** [comment]
I agree.  We'll hammer on this in testing the build, and if it doesn't repro, close out.
